velox/type/DecimalUtil.h

```diff
@@ -32,5 +32,12 @@ class DecimalUtil {
   /// Helper function to convert a decimal value to string.
   template <typename T>
   static std::string toString(const T& value, const TypePtr& type);
+
+  template <class T>
+  static void rescaleWithRoundUp(
```

Comment:
This implementation is performance critical.
Let's put the implementations in the header so that the compiler will be able to inline them.

Reply:
Done.